lib/matplotlib/tests/test_axes.py

```diff
@@ -4972,3 +4972,21 @@ def test_bar_single_height():
     ax.bar(range(4), 1)
     # Check that a horizontal chart with one width works
     ax.bar(0, 1, bottom=range(4), width=1, orientation='horizontal')
+
+def test_datetime_axhline_same_axes():
```

Comment:
Looks like you'll also need to trim trailing whitespace: 
https://travis-ci.org/matplotlib/matplotlib/jobs/208246492#L1683